requirements/customer/cli.tex

```diff
@@ -0,0 +1,102 @@
+\newcommand{\jacommand}[3]{
+
+\textsc{\large{\textbf{#1}}}
+\begin{itemize}
+\item[\textbf{Options:}] #2
+\item[\textbf{Effect:}] #3
+\end{itemize}
+}
+\newcommand{\jaoption}[3]{
+\begin{itemize}
+\item[] \textbf{#1} (#2)
+\item[\textbf{Effect:}] #3
+\end{itemize}
+}
+\subsection{Command Line Interface}
+All components of JobAdder are accessible through a command line interface (CLI).
+The command line interface on a machine with JobAdder installed has the following structure:
+\begin{equation}
+\text{<component name> <command> <options> <targets>},
+\end{equation}
+where the \textit{component name} can be any one of the following:
+\begin{itemize}
+\item \textbf{jobadder} to access the \textit{user client}.
+\item \textbf{jobcenter} to access the \textit{server}.
+\item \textbf{jobworker} to access the \textit{worker client}.
+\end{itemize}
+Commands, options, and valid targets are and options are specified in the following sections.
+
+Options can be added in the following ways:
+\begin{itemize}
+\item By supplying their names like "--<option\_x> --<option\_y>=2 --<option\_z>".
```

Comment:
I think there is a problem with LaTeX formatting on this line and below. It interprets the `--` as one long line.
Maybe we should use something like the solutions described here: https://tex.stackexchange.com/questions/286094/insert-code-keywords-inline ?

Reply:
In this case there is a simple solution: You can just replace `--` with `-{}-` so that it's no longer being interpreted as a TeX ligature.
However, depending on how often we plan to use code snippets in our document it might be worthwhile to use a package for properly displaying source code.